### PR TITLE
Apply Licensing Info for Strict Auditing

### PR DESCRIPTION
**Pull request by sschwetz**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/adplug.rb [hunk 1]
```diff
@@ -3,7 +3,8 @@ class Adplug < Formula
   homepage "https://adplug.github.io"
   url "https://github.com/adplug/adplug/releases/download/adplug-2.3.3/adplug-2.3.3.tar.bz2"
   sha256 "a0f3c1b18fb49dea7ac3e8f820e091a663afa5410d3443612bf416cff29fa928"
-  license "LGPL-2.1"
+  license "LGPL-2.1-or-later"
+  revision 1
```
Comment (carlocab) [inline Formula/adplug.rb:7]:
```suggestion
```

File: Formula/advancecomp.rb [hunk 2]
```diff
@@ -3,8 +3,8 @@ class Advancecomp < Formula
   homepage "https://www.advancemame.it/comp-readme.html"
   url "https://github.com/amadvance/advancecomp/releases/download/v2.1/advancecomp-2.1.tar.gz"
   sha256 "3ac0875e86a8517011976f04107186d5c60d434954078bc502ee731480933eb8"
-  license "GPL-3.0"
-  revision 1
+  license "GPL-3.0-or-later"
+  revision 2
```
Comment (carlocab) [inline Formula/advancecomp.rb:7]:
```suggestion
  revision 1
```

File: Formula/advancemame.rb [hunk 3]
```diff
@@ -3,7 +3,8 @@ class Advancemame < Formula
   homepage "https://www.advancemame.it/"
   url "https://github.com/amadvance/advancemame/releases/download/v3.9/advancemame-3.9.tar.gz"
   sha256 "3e4628e1577e70a1dbe104f17b1b746745b8eda80837f53fbf7b091c88be8c2b"
-  license "GPL-2.0"
+  license "GPL-2.0-only"
+  revision 1
```
Comment (carlocab) [inline Formula/advancemame.rb:7]:
```suggestion
```

File: Formula/aha.rb [hunk 4]
```diff
@@ -3,7 +3,8 @@ class Aha < Formula
   homepage "https://github.com/theZiz/aha"
   url "https://github.com/theZiz/aha/archive/0.5.1.tar.gz"
   sha256 "6aea13487f6b5c3e453a447a67345f8095282f5acd97344466816b05ebd0b3b1"
-  license "LGPL-2.1"
+  license "LGPL-2.1-only"
+  revision 1
```
Comment (carlocab) [inline Formula/aha.rb:7]:
```suggestion
```

File: Formula/advancescan.rb [hunk 5]
```diff
@@ -3,7 +3,8 @@ class Advancescan < Formula
   homepage "https://www.advancemame.it/scan-readme.html"
   url "https://github.com/amadvance/advancescan/releases/download/v1.18/advancescan-1.18.tar.gz"
   sha256 "8c346c6578a1486ca01774f30c3e678058b9b8b02f265119776d523358d24672"
-  license "GPL-2.0"
+  license "GPL-2.0-only"
+  revision 1
```
Comment (carlocab) [inline Formula/advancescan.rb:7]:
```suggestion
```

File: Formula/advancemenu.rb [hunk 6]
```diff
@@ -3,7 +3,8 @@ class Advancemenu < Formula
   homepage "https://www.advancemame.it/menu-readme.html"
   url "https://github.com/amadvance/advancemame/releases/download/v3.9/advancemame-3.9.tar.gz"
   sha256 "3e4628e1577e70a1dbe104f17b1b746745b8eda80837f53fbf7b091c88be8c2b"
-  license "GPL-2.0"
+  license "GPL-2.0-only"
+  revision 1
```
Comment (carlocab) [inline Formula/advancemenu.rb:7]:
```suggestion
```

**Review by carlocab — CHANGES_REQUESTED** [review]
Please don't bump the `revision`s for license-only changes.

**Comment by carlocab** [comment]
Thanks for the PR, @sschwetz. Do you have references for these being `*-only` rather than `*-or-later`, or vice-versa? These are usually a little tricky to determine.

It would make this a lot easier to review if you split this PR up so that each of your PRs modifies only 1-3 formulae each.

**Comment by sschwetz** [comment]
I  went through the code repositories / websites and looked for licensing terms. For sites that specifically mentioned that they were licence version x or later than they got the or-later. Any other repos v3 with no information about moving to newer  licences were marked as  -only.

For GPL-2 I marked as -only if they did not specify V3 due to the issues that are seen with V3

I will seperate further pulls into smaller batches

**Comment by carlocab** [comment]
Sorry, this requires a bit of time to review because it modifies a lot of formulae. I'll have a look again at this soon.

**Pull request closed without merging**